packages/api-client/src/components/ScalarHotkey.vue

```diff
@@ -1,42 +1,27 @@
 <script setup lang="ts">
-import { keyMap } from '@/hooks'
+import type { HotKeyModifiers } from '@scalar/oas-utils/entities'
 import { isMacOS } from '@scalar/use-tooltip'
-import { useMagicKeys, whenever } from '@vueuse/core'
 import { computed } from 'vue'
 
 const props = defineProps<{
-  hotkey: string
+  hotkey: {
```

Comment:
I was thinking more like:
```suggestion
  hotkey: string
  modifier?: HotKeyModifiers
```

(As individual props instead of an object)

Then you don't have to change how the component is used in all those places

Reply:
i agree, good call!